bodhi/templates/update.html

```diff
@@ -535,6 +535,8 @@
             % endif
           % elif update.meets_testing_requirements and getattr(update.request, 'description', None) != 'stable':
             <a id='stable' class="btn btn-sm btn-success"><span class="glyphicon glyphicon-circle-arrow-right"></span> Push to Stable</a>
+          % elif update.stable_karma not in (0, None) and update.karma >= update.stable_karma and not update.autokarma and getattr(update.request, 'description', None) != 'stable':
```

Comment:
The autopush only triggers when the karma goes from 0 to 1, not when it is already at 1, that's the difference.